postmark/assets/logs/postmark.yaml

```diff
@@ -0,0 +1,328 @@
+          sourceType: attribute
+          target: http.url
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: url-parser
+          name: Extracting url-details from the url
+          enabled: true
+          sources:
+            - http.url
+          target: http.url_details
+          normalizeEndingSlashes: false
+        - type: attribute-remapper
+          name: Map `Geo.IP` to `network.client.ip`
+          enabled: true
+          sources:
+            - Geo.IP
+          sourceType: attribute
+          target: network.client.ip
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: geo-ip-parser
+          name: Define `network.client.ip` as default geoip attribute for source
+          enabled: true
+          sources:
+            - network.client.ip
+          target: network.client.geoip
+          ip_processing_behavior: do-nothing
+        - type: attribute-remapper
+          name: Map `UserAgent` to `http.useragent`
+          enabled: true
+          sources:
+            - UserAgent
+          sourceType: attribute
+          target: http.useragent
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: user-agent-parser
+          name: Extracting user-agent information from the user-agent
+          enabled: true
+          sources:
+            - http.useragent
+          target: http.useragent_details
+          encoded: false
+          combineVersionDetails: false
+    - type: pipeline
+      name: Processing for subscription change events
+      enabled: true
+      filter:
+        query: "@evt.name:SubscriptionChange"
+      processors:
+        - type: attribute-remapper
+          name: Map `ChangedAt` to `timestamp`
+          enabled: true
+          sources:
+            - ChangedAt
+          sourceType: attribute
+          target: timestamp
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - name: Lookup for `SuppressSending` to `status` field
+          enabled: true
+          source: SuppressSending
+          target: status
+          lookupTable: true,warning
+          type: lookup-processor
+    - type: date-remapper
+      name: Define `timestamp` as the official date of the log
+      enabled: true
+      sources:
+        - timestamp
+    - type: attribute-remapper
+      name: Map `Recipient` to `usr.email`
+      enabled: true
+      sources:
+        - Recipient
+      sourceType: attribute
+      target: usr.email
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - name: Lookup for `evt.name` to `status` field
+      enabled: true
+      source: evt.name
+      target: status
+      lookupTable: |-
+        Bounce,error
+        SpamComplaint,warning
+        Delivery,Success
```

Comment:
Just to confirm: `Open` and `Click` events do not have entries here because you want them to map to `Info`?

Reply:
Yes, that’s correct.